less/index.less

```diff
@@ -57,3 +57,171 @@ html, body {
     height: 100%;
 }
 
+/* Start of CSS */
+
+@borderColor: blue;
+
+/*  Mixins */
+.flexbox-mixin {
+  display: -webkit-box;
+  display: -moz-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.secTwo-Mixin {
+	margin-top: 5%;
+			height: 300px;
+			width: 300px;
+			
+}
+
+.pictureBlock-mixin {
+	height: 300px;
+	width: 33%;
+	margin-top: 4%;
+
+}
+
+/* End Mixins */
+
+.container {
+	margin: 0 15% 0 15%;
+	height: 100%;
+
+	
+	html {
+		font-size: 62.5%;
+		
+	}
+
+	nav {
+		.flexbox-mixin();
+		justify-content: center;
+		flex-direction: row;
+		align-content: space-evenly;
+	}
+
+	 a{
+		.flexbox-mixin();
+		font-size: 1.2rem;
+		flex-direction: row;
+		margin: 1%;
+		
+	}
+
+	.pictureBlock {
+		.flexbox-mixin();		
+		flex-direction: row;
+		justify-content: space-between;
+		background-color: lightgrey;
+		margin-top: 5%;
+		height: 45%;
+		width: 100%;
+
+		.blackBox {
+			margin-left: 5%;
+			.pictureBlock-mixin();
+			color: yellow;
+			.flexbox-mixin();
+			justify-content: center;
+			align-items: center;
+			background-color: black;
+		}
+
+		.intro {
+			.pictureBlock-mixin();
+			margin-right: 5%;
+			
+			
+	
+		}
+	}
+
+	.sectionTwo {
+		.flexbox-mixin();
+		justify-content: space-evenly;
+		margin-top: 5%;
+		height: 50%;
+		width: 100%;
+		
+
+		.skills {
+			.secTwo-Mixin();
+
+				ol {
+					.flexbox-mixin();
+					flex-direction: column;
+					justify-content: space-evenly;
+					align-items: flex-start;
+					margin-top: 15%;
+				}
+
+				li {
+					margin: 5%;
+				}
+
+				p {
+					margin-top: 5%;
+					margin-left: 5%;
+					
+				}
+		}
+
+		.workHistory {
+			.secTwo-Mixin();
+
+				ol {
+					.flexbox-mixin();
+					flex-direction: column;
+					justify-content: space-evenly;
+					align-items: flex-start;
+					margin-top: 15%;
```

Comment:
Avoid using percentages for vertical spaces (margin/padding top & bottom, height)